internal/clients/connect/content.go

```diff
@@ -83,3 +102,48 @@ func ConnectContentFromConfig(cfg *config.Config) *ConnectContent {
 	}
 	return c
 }
+
+func ConnectContentSummaryFromServer(account *accounts.Account, log logging.Logger) ([]*ConnectContentSummary, error) {
+	client, err := NewConnectClient(account, 30*time.Second, events.NewNullEmitter(), log)
+	if err != nil {
+		return nil, err
+	}
+	contentItems, err := client.GetContent(log)
+	if err != nil {
+		return nil, err
+	}
+	count := 0
+	for _, content := range *contentItems {
+		if content.Role == "owner" || content.Role == "editor" {
+			count += 1
```

Comment:
if you want to avoid a pre-counting pass, you can allocate an empty `summary` slice and then `append` to it.